src/RSCPL.xml

```diff
@@ -0,0 +1,473 @@
+             a license thereto.</p>
+        </li>
+        <li>
+          <b>1.12.</b>
+          <p>"RSV" means Ricoh Silicon Valley, Inc., a California corporation with offices at 2882 Sand Hill
+             Road, Suite 115, Menlo Park, CA 94025-7022.</p>
+        </li>
+        <li>
+          <b>1.13.</b>
+          <p>"Source Code" means the preferred form of the Governed Code for making modifications to it,
+             including all modules it contains, plus any associated interface definition files, scripts
+             used to control compilation and installation of Executable Code, or a list of source code
+             differential comparisons against either the Original Code or another well known, available
+             Governed Code of the Contributor's choice. The Source Code can be in a compressed or archival
+             form, provided the appropriate decompression or de-archiving software is widely available for
+             no charge.</p>
+        </li>
+        <li>
+          <b>1.14.</b>
+          <p>"You" means an individual or a legal entity exercising rights under, and complying with all of
+             the terms of, this License or a future version of this License issued under Section 6.1. For
+             legal entities, "You" includes any entity which controls, is controlled by, or is under common
+             control with You. For purposes of this definition, "control" means (a) the power, direct or
+             indirect, to cause the direction or management of such entity, whether by contract or
+             otherwise, or (b) ownership of fifty percent (50%) or more of the outstanding shares or
+             beneficial ownership of such entity.</p>
+        </li>
+        </list>
+        </li>
+        <li>
+          <b>2.</b>
+          <p>Source Code License.</p>
+      <list>
+        <li>
+          <b>2.1.</b>
+          <p>Grant from RSV. RSV hereby grants You a worldwide, royalty-free, non-exclusive license, subject
+             to third party intellectual property claims:</p>
+      <list>
+        <li>
+          <b>(a)</b>
+          <p>to use, reproduce, modify, create derivative works of, display, perform, sublicense and
+             distribute the Original Code (or portions thereof) with or without Modifications, or as part
+             of a Larger Work; and</p>
+        </li>
+        <li>
+          <b>(b)</b>
+          <p>under Patent Claims infringed by the making, using or selling of Original Code, to make, have
+             made, use, practice, sell, and offer for sale, and/or otherwise dispose of the Original Code
+             (or portions thereof).</p>
+        </li>
+        </list>
+        </li>
+        <li>
+          <b>2.2.</b>
+          <p>Contributor Grant. Each Contributor hereby grants You a worldwide, royalty-free, non-exclusive
+             license, subject to third party intellectual property claims:</p>
+      <list>
```

Comment:
This (and some of the subsequent lines) could use deeper indentation to show that they're part of the 2.2 list entry.